wordpress-mobile/WordPress-Android
# Fix disappearing saved posts

Fixes #8791 

The issue is caused by a bug which has been in the app for ages, it just hadn't had any user-facing impact until we implemented Saved Posts.

We keep only 200 most recent posts for each tag in the cache. However, there is a bug in the SQL query so we are not removing just posts which don't fit in first 200, but also all posts which have different tag than the one currently selected. 

To test
1. Open Reader - followed sites tab
2. Scroll down as far as you can (there is a limit)
3. Save the last post in the list
4. Go to Saved posts and notice the post is there
5. Go to Discover tab
6. Start following one of the sites
7. Go back to Followed-sites tab
8. Restart the app
9. Go to Saved posts and notice the post is still there

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

cc @kwonye
